container/rules/docker_toolchains.bzl

```diff
@@ -139,7 +139,7 @@ def _language_tool_layer_impl(ctx, symlinks=None, env=None, tars=None,
     ctx.actions.run_shell(
       inputs=installables_tars,
       outputs=[final_installables_tar],
-      command="tar cvf {output_tar} --files-from /dev/null && tar --concatenate -v --file={output_tar} {input_tars}".format(
+      command="tar cvf {output_tar} --files-from /dev/null && for i in {input_tars}; do tar --concatenate --file={output_tar} $i; done".format(
```

Comment:
Can we wrap this line. Also, could you comment on the PR comment what was the issue with how this was done before?

Reply:
Done. Added comments as well.